data/json/snippets/addiction.json

```diff
@@ -112,7 +118,12 @@
       "Your hands start shaking… you need some painkillers.",
       "If you don't find some opiates soon you might start engaging in reckless behavior, you think to yourself.",
       "You daydream of crushing pills and snorting them through a rolled up dollar bill.  The drip in the back of your throat presaging a fun time.",
-      "If I can just find some oxies, everything will be fine."
+      "If I can just find some oxies, everything will be fine.",
+      "You can imagine it right now, that tingle starting at your extremities, warm and welcoming.  But it's not real, you haven't had any opiates today.  Why haven't you?  The world already ended, you don't have to stay sober to talk to your family or a job.",
+      "Today is a good day for hydros.",
```

Comment:
```suggestion
      "Today is a good day for dillies.",
```

Reply:
Hydros is slang for acetaminophen and codeine. 
I didn’t think we had dilaudid in game?

Reply:
I have never heard that one. Mostly there's no slang for that, it's used on the street but it's not that popular as a street drug. People call it t3 or tylenol

Reply:
I see that you are unfamiliar with the works of 3 6 Mafia.

Reply:
Definitely was in slang in the 90's to mid 2000's here's one of their many lyrical references to it.  https://www.youtube.com/watch?v=XTBWZw0iUp0

Might just be a USA thing.

Reply:
So you made me question was this just a southern thing, and I was like no surely Bone Thugs N Harmony also referred to them as hydros.  Yes, yes they did. https://www.youtube.com/watch?v=oXykr1BnveQ  Thank you for making me check my long held assumptions.